apps/explorer/src/ui/image/Image.tsx

```diff
@@ -58,10 +61,18 @@ function BaseImage({
     src,
     srcSet,
     fit,
-    blur,
+    visibility,
     onClick,
     ...imgProps
 }: ImageProps & { status: string }) {
+    const [isBlurred, setIsBlurred] = useState(false);
```

Comment:
Doesn't need to be state, could just use visibility directly.

Reply:
This state is here so that the user can unblur images